parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_hub_rococo_config.rs

```diff
@@ -46,6 +46,7 @@ parameter_types! {
 	pub const MaxUnconfirmedMessagesAtInboundLane: bp_messages::MessageNonce =
 		bp_bridge_hub_rococo::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
 	pub const BridgeHubWococoChainId: bp_runtime::ChainId = bp_runtime::BRIDGE_HUB_WOCOCO_CHAIN_ID;
+	pub BridgeWococoMessagesPalletInstance: Option<InteriorMultiLocation> = Some(X1(PalletInstance(<BridgeWococoMessages as PalletInfoAccess>::index() as u8)));
```

Comment:
```suggestion
	pub BridgeWococoMessagesPalletInstance: InteriorMultiLocation = X1(PalletInstance(<BridgeWococoMessages as PalletInfoAccess>::index() as u8));
```

This should work, too.

Reply:
thank you, both changed